webview/src/plots/components/GetStarted.tsx

```diff
@@ -5,7 +5,6 @@ import { StartButton } from '../../shared/components/button/StartButton'
 
 export type AddPlotsProps = {
   hasUnselectedPlots: boolean
-  hasSelectedRevisions: boolean
```

Comment:
I noticed this `hasSelectedRevisions` isn't being used anywhere so I deleted it. Or is it supposed to be used to hide the `Add Experiments` button?

Reply:
Looks like it was a simple [leftover](https://github.com/iterative/vscode-dvc/pull/2932/files#diff-45b20cc7f55a9ad8b30d0c84017251735c1a0bfa97a4ef787da36c2d7ecb96c3). Good catch!